content/en/docs/started/support.md

```diff
@@ -4,224 +4,99 @@ description = "Where to get support for Kubeflow"
+  * [Kserve](https://github.com/kserve/kserve/issues)
+  * [Platform / Manifests](https://github.com/kubeflow/manifests/issues)
   * [Examples](https://github.com/kubeflow/examples/issues)
-  * [Documentation](https://github.com/kubeflow/website/issues)
+  * [Website](https://github.com/kubeflow/website/issues)
 
 <a id="provider-support"></a>
-## Support from providers in the Kubeflow ecosystem
-
-The following organizations offer advice and support for Kubeflow deployments:
-
-<div class="table-responsive">
-  <table class="table table-bordered">
-    <thead class="thead-light">
-      <tr>
-        <th>Organization</th>
-        <th>Points of contact</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr>
-        <td><a href="https://www.arrikto.com">Arrikto</a></td>
-        <td><a href="mailto:sales@arrikto.com">sales@arrikto.com</a></td>
-      </tr>
-      <tr>
-        <td><a href="https://www.ubuntu.com">Canonical</a></td>
-        <td><a href="https://ubuntu.com/kubeflow#get-in-touch">Get in touch</a></td>
-      </tr>
-      <tr>
-        <td><a href="https://www.pattersonconsultingtn.com/">Patterson Consulting</a></td>
-        <td> 
-        <a href="http://www.pattersonconsultingtn.com/offerings/managed_kubeflow.html">Managed Kubeflow</a></td>
-      </tr>
-      <tr>
-        <td><a href="https://www.seldon.io/">Seldon</a></td>
-        <td> 
-        <a href="https://github.com/SeldonIO/seldon-core/issues">Issue 
-        tracker</a></td>
-      </tr>    
-    </tbody>
-  </table>
-</div>
+## Support from commercial providers in the Kubeflow Ecosystem
 
-<a id="cloud-support"></a>
-## Support from a cloud or platform provider
+We want to promote commercial companies and idividuals that contribute back to the open source project.
+Below is a table of organizations that contribute to Kubeflow and offer commercial support:
 
-If you're using the services of a cloud provider to host Kubeflow, the cloud
-provider may be able to help you diagnose and solve a problem.
-
-Consult the support page for the cloud service or platform that you're using:
-
-* [Amazon Web Services (AWS)](https://aws.amazon.com/contact-us/)
-* [Canonical Ubuntu](https://ubuntu.com/kubeflow#get-in-touch)
-* [Google Cloud Platform (GCP)](https://cloud.google.com/support-hub/)
-* [IBM Cloud](https://www.ibm.com/cloud/support)
-* [Microsoft Azure](https://azure.microsoft.com/en-au/support/options/)
-* [Red Hat OpenShift](https://help.openshift.com/)
-
-## Other places to ask questions
+| Provider               | Support Link                                                   |
+|------------------------|----------------------------------------------------------------|
+| Canonical              | [Ubuntu Kubeflow](https://ubuntu.com/kubeflow#get-in-touch)    |
+| Aranui Solutions       | [Aranui Solutions](https://...)                                |
+| Freelance/Consulting   | Julius von Kohout on the Slack channel                         |
```

Comment:
I am pro listing individual people that have contributed here. 

Probably this opens up a rabbithole for us to have some sort of vetting process or disclaimer .

Reply:
The same goes for each company. They should have a track record of freelancers / employees contributing to kubeflow

Reply:
The commercial providers section was also there before. I just cleaned it up. I think it is fair if we just do alphabetical sorting or so and only list ompanies  / individuals that have significant contributions in the last 1-2 years or so. If in doubt we can ask the KSC and use the first batch as reference.

Reply:
i added all link. I think it is good enough to significantly improve the status quo